src/post_section/post_section.jsx

```diff
@@ -30,102 +31,63 @@ function LinksList({ link }) {
-                            know what in mind,
-                            considering that sometimes totally bonkers stuff might be highly relevant if it has the right kind
-                            of
-                            thinking behind it. </p>
-                        <h2 className="pipeline">Next on the pipeline</h2>
-                        <p className="text_after_pipeline text_of_section">This lovely web is full of everything which is created I
-                            don't know what in
-                            mind, considering
-                            that
-                            sometimes totally bonkers stuff might be highly relevant if it has the right kind of thinking behind
-                            it.
-                        </p>
+                        <table className="table_with_photo">
+                            <tr className="table_of_contact">
+                                {links.map((link) => (<LinksList link={link} />))}
+                            </tr>
+                        </table>
                     </div>
-                    <div className="rectangle">
-                        <picture>
-                            <source media="(max-width:854px)" width="375" height="250" srcset="/photos/rectangle.png" />
-                            <source media="(min-width:854px)" width="854" height="570" srcset="/photos/rectangle_desktop.jpg" />
-                            <img src="/photos/rectangle_desktop.jpg" width="854" height="570" alt="rectangle" />
-                        </picture>
-                        <figcaption className="rectangle_caption">Image caption centered this way and I’ll make this a bit longer to
-                            indicate the amount of line-height</figcaption>
+                </div>
+                <div className="rectangle">
+                    <picture>
+                        <source media="(max-width:854px)" width="375" height="250" srcset={data.posts.thumbnail_url} />
+                        <source media="(min-width:854px)" width="854" height="570" srcset={data.posts.thumbnail_url} />
+                        <img src={data.posts.thumbnail_url} width="854" height="570" alt="main photo" />
+                    </picture>
+                    <figcaption className="rectangle_caption">{data.posts.title}</figcaption>
+                </div>
+                <div className="second_section_block">
+                    {<div className='post_body' dangerouslySetInnerHTML={{ __html: data.posts.content }} />}
+                    <div className="article_section_footer">
+                        <p className="section_footer_text share">Share:</p>
+                        <table className="table_footer">
+                            <tr className="table_of_contact_second">
+                                {links.map((link) => (<LinksListBottom link={link} />))}
```

Comment:
again, component is called `LinkListBottom` but it renders a single link. rename